### PR TITLE
Fix CI Failures

### LINKED ISSUE #26: [Bug]: CI is failing with Ruff and Bandit Issues
### Descriription

Some code from before the code quality tools were standardized is causing CI to fail. 

### Expected behavior

CI passes.

### Steps to reproduce the behavior

# Locally

Run: 

```
❯ ruff check .
src/web/views.py:8:12: F821 Undefined name `render`
Found 1 error.
```

and 

```
❯ bandit -c pyproject.toml -r .
[main]	INFO	profile include tests: None
[main]	INFO	profile exclude tests: None
[main]	INFO	cli include tests: None
[main]	INFO	cli exclude tests: None
[main]	INFO	using config: pyproject.toml
[main]	INFO	running on Python 3.11.7
Run started:2024-02-02 05:14:34.449627

Test results:
>> Issue: [B105:hardcoded_password_string] Possible hardcoded password: 'django-insecure-t9*!4^fdn*=pmz4%8u_we!88e!8@_!drx0)u_@6$@!nx$4svjp'
   Severity: Low   Confidence: Medium
   CWE: CWE-259 (https://cwe.mitre.org/data/definitions/259.html)
   More Info: https://bandit.readthedocs.io/en/1.7.6/plugins/b105_hardcoded_password_string.html
   Location: ./src/spokanetech/settings.py:28:17
27	   # SECURITY WARNING: keep the secret key used in production secret!
28	   SECRET_KEY = "django-insecure-t9*!4^fdn*=pmz4%8u_we!88e!8@_!drx0)u_@6$@!nx$4svjp"
29	

--------------------------------------------------

Code scanned:
	Total lines of code: 211
	Total lines skipped (#nosec): 0

Run metrics:
	Total issues (by severity):
		Undefined: 0
		Low: 1
		Medium: 0
		High: 0
	Total issues (by confidence):
		Undefined: 0
		Low: 0
		Medium: 1
		High: 0
Files skipped (0):
```

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by KGB33**
## Pull Request

**Description:**
Minor code fixes to pass CI. 

Also fixes 500 error on live site.

20240201_21h28m55s_grim

**Related Issues:**
- Closes #26 

**Checklist:**
- [x] Code follows the project's coding standards.
- [ ] All tests pass.
- [ ] Documentation has been updated.

### CODE REVIEW

File: src/spokanetech/settings.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 IS_DEVELOPMENT = bool(os.environ.get("SPOKANE_TECH_DEV", False))
 if IS_DEVELOPMENT:
     # SECURITY WARNING: keep the secret key used in production secret!
-    SECRET_KEY = "django-insecure-t9*!4^fdn*=pmz4%8u_we!88e!8@_!drx0)u_@6$@!nx$4svjp"
+    SECRET_KEY = "django-insecure-t9*!4^fdn*=pmz4%8u_we!88e!8@_!drx0)u_@6$@!nx$4svjp"  # nosec: Development-only key.
```
Comment (joeriddles) [inline src/spokanetech/settings.py:28]: neat!

**Review by joeriddles — APPROVED** [review]
Thank you!

**Comment by joeriddles** [comment]
Looks like the CI workflows aren't queuing up.
